source/libraries/CompileSolidity.ts

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as solc from 'solc';
+
+class SolidityContractCompiler {
+    private compiler: any = solc;
```

Comment:
No need to have a private member that is just an alias to the import.  Recommend deleting this line and just using `solc` below.  If desired, you could change the import line to `import * as compiler from 'solc'` if you find the terminology of `compiler` to be more clear.

Personally, I like `solc` as it is more explicit but I'll leave the naming decision here up to you.

Reply:
Ok, `solc` is fine with me since it's more explicit.